libs/net/dal/Services/ContentService.cs

```diff
@@ -175,6 +175,24 @@ public IPaged<Content> FindWithDatabase(ContentFilter filter, bool asNoTracking
         return new Paged<Content>(items, filter.Page, filter.Quantity, total);
     }
 
+    /// <summary>
+    /// Find today's front pages
+    /// </summary>
+    /// <returns>The 5 front pages for today's date.</returns>
+    public IPaged<Content> FindFrontPages(bool asNoTracking = true)
+    {
+        var query = this.Context.Contents
+            .Include(c => c.FileReferences)
+            .Where(c => c.ProductId == 10)
```

Comment:
If this is not overkill - I am guessing this should have an enum instead of just the int?